scss/_stepped-process.scss

```diff
@@ -0,0 +1,139 @@
+//
+// Orange stepped process
+//
+.stepped-process {
+  ol {
+    display: flex;
+    padding: 0;
+    margin: 0;
+    overflow: hidden;
+    counter-reset: step;
+  }
+}
+
+.stepped-process-item {
+  display: flex;
+  flex: 1;
+  padding: map-get($spacers, 2);
+  margin-right: $stepped-process-margin-r;
+  font-weight: $font-weight-bold;
+  text-align: center;
+  list-style: none;
+  counter-increment: step;
+  background-color: $black;
+}
+
+.current {
+  background-color: $primary;
+
+  .stepped-process-link {
+    &,
+    &:hover,
+    &:focus,
+    &:active {
+      color: $black;
+    }
+  }
+
+  ~ .stepped-process-item {
+    background-color: $gray-400;
+
+    .stepped-process-link {
+      &,
+      &:hover,
+      &:focus,
+      &:active {
+        color: $black;
+      }
+    }
+  }
+}
```

Comment:
I think all of this could be simplified by using custom properties: I'll give it a shot to show you.